include/CommandBasedBackend.h

```diff
@@ -60,7 +60,7 @@ namespace ChimeraTK {
     std::vector<std::string> sendCommand(std::string cmd, size_t nLinesExpected);
 
     template<typename UserType>
-    boost::shared_ptr<NDRegisterAccessor<UserType>> getRegisterAccessor_impl(
+    boost::shared_ptr<NDRegisterAccessor<UserType>> getRegisterAccessor_impl( // NOLINT
```

Comment:
Generally disabling the linter makes us blind against other mistakes in that line or block of code.
If is possible to only disable certain messages:
```suggestion
    // NOLINTNEXTLINE(readability-identifier-naming)
    boost::shared_ptr<NDRegisterAccessor<UserType>> CommandBasedBackend::getRegisterAccessor_impl(
```